From c0dba2cca4d2bf3526d90a2050bdb17148ce803f Mon Sep 17 00:00:00 2001
From: Patrick Steuer <psteuer@mail.de>
Date: Sat, 15 Oct 2016 17:41:41 +0200
Subject: [PATCH] Fix strict-warnings build

crypto/s390xcap.c: internal/cryptlib.h needs to be included for
OPENSSL_cpuid_setup function prototype is located there to avoid
build error due to -Werror=missing-prototypes.

Signed-off-by: Patrick Steuer <psteuer@mail.de>

Reviewed-by: Rich Salz <rsalz@openssl.org>
Reviewed-by: Richard Levitte <levitte@openssl.org>
Reviewed-by: Matt Caswell <matt@openssl.org>
CLA: trivial
---
 crypto/s390xcap.c | 1 +
 1 file changed, 1 insertion(+)

--- a/crypto/s390xcap.c
+++ b/crypto/s390xcap.c
@@ -12,6 +12,7 @@
 #include <string.h>
 #include <setjmp.h>
 #include <signal.h>
+#include "internal/cryptlib.h"
 
 unsigned long long OPENSSL_s390xcap_P[10];
 
